kong/router/atc.lua

```diff
@@ -516,7 +481,7 @@ function _M:exec(ctx)
   CACHE_PARAMS.uri  = req_uri
   CACHE_PARAMS.host = req_host
 
-  local cache_key = get_cache_key(self.fields, CACHE_PARAMS)
+  local cache_key = self.fields:get_cache_key(CACHE_PARAMS)
```

Comment:
We can `local fields = self.fields` at the beginning of the function to speed up this critical function call.

Reply:
No, `self.fields` is called only once in `matching()` and `exec()`, I think that localization of it is not very helpful.

Reply:
Add localization in stream's `exec()`.